installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -66,6 +66,12 @@ include::modules/ssh-agent-using.adoc[leveloffset=+1]
 include::modules/installation-user-infra-generate.adoc[leveloffset=+1]
 include::modules/installation-minimum-resource-requirements.adoc[leveloffset=+2]
 include::modules/installation-aws-tested-machine-types.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features] in the AWS documentation for more information about AWS Local Zones and the supported instances types and services. 
```

Comment:
Per the mod docs/jupiter templates, additional resources should just be a link if possible (not a sentence / additional text).

Though I was wondering myself recently for my own PR about specifying if it's a link outside of RH. @kalexand-rh do you know if it's okay to say "in the <x> documentation" after a link if it's outside of RH? Or should we still leave that off, per jupiter requirements?